src/main/kotlin/no/nav/tilleggsstonader/sak/opplysninger/ansvarligSaksbehandler/domain/AnsvarligSaksbehandler.kt

```diff
@@ -0,0 +1,24 @@
+package no.nav.tilleggsstonader.sak.opplysninger.ansvarligSaksbehandler.domain
+
+import no.nav.tilleggsstonader.sak.opplysninger.ansvarligSaksbehandler.dto.AnsvarligSaksbehandlerDto
+
+data class AnsvarligSaksbehandler(
+    val fornavn: String?,
+    val etternavn: String?,
+    val rolle: SaksbehandlerRolle,
+)
+
+enum class SaksbehandlerRolle {
```

Comment:
Skulle vi gå vekk fra å bruke "rolle"?
```suggestion
enum class AnsvarligSaksbehandlerStatus {
```